plugins/content/attachment/lib/api/filters/KalturaAttachmentAssetFilter.php

```diff
@@ -4,7 +4,33 @@
  * @subpackage api.filters
  */
 class KalturaAttachmentAssetFilter extends KalturaAttachmentAssetBaseFilter
-{
+{	
+	/**
+	 * @dynamicType KalturaAssetType
+	 * @var string
+	 */
+	public $typeNotIn;
```

Comment:
It's a bit problematic to put `typeNotIn` on `KalturaAttachmentAssetFilter`, type is a property of Asset.
That's why we wanted to add a new type to allow filtering by "not in".